assemble/src/main/resources/LICENSE

```diff
@@ -1,982 +0,0 @@
-
```

Comment:
That clears everything up.  Thanks.  Looking at the first few lines, I had assumed it was just an Apache license file.  After reading your comment I looked more closely and saw what it really was.